packages/backend/src/models/GroupsModel.ts

```diff
@@ -155,30 +155,33 @@ export class GroupsModel {
     }
 
     async createGroup(
-        group: CreateGroup & { organizationUuid: string },
-    ): Promise<Group> {
+        createGroup: CreateGroup & { organizationUuid: string },
+    ): Promise<GroupWithMembers> {
         const results = await this.database.raw<{
             rows: { created_at: Date; group_uuid: string }[];
         }>(
             `
-            INSERT INTO groups (name, organization_id)
-            SELECT ?, organization_id
-            FROM organizations
-            WHERE organization_uuid = ?
-            RETURNING group_uuid, groups.created_at
-        `,
-            [group.name, group.organizationUuid],
+                INSERT INTO groups (name, organization_id)
+                SELECT ?, organization_id
+                FROM organizations
+                WHERE organization_uuid = ?
+                RETURNING group_uuid, groups.created_at
+            `,
+            [createGroup.name, createGroup.organizationUuid],
         );
         const [row] = results.rows;
         if (row === undefined) {
-            throw new UnexpectedDatabaseError(`Failed to create organization`);
+            throw new UnexpectedDatabaseError(`Failed to create group`);
         }
-        return {
-            uuid: row.group_uuid,
-            name: group.name,
-            createdAt: row.created_at,
-            organizationUuid: group.organizationUuid,
-        };
+
+        if (createGroup.members && createGroup.members.length > 0) {
+            await this.addGroupMembers(
+                row.group_uuid,
+                createGroup.members.map((m) => m.userUuid),
```

Comment:
can't we just use `createGroup.memberUuids` here?

Reply:
`CreateGroup` argument type doesn't have that property.